soot-infoflow-android/src/soot/jimple/infoflow/android/manifest/BaseProcessManifest.java

```diff
@@ -516,6 +516,35 @@ public Set<String> getPermissions() {
 	}
 	
 	
+ /** *
+     * polyu zachary
```

Comment:
Please improve the JavaDoc. The description should be in English, and should properly describe what the method does.